packaging/install-prestoadmin.template

```diff
@@ -61,3 +61,12 @@ mkdir -p $LOG_DIR
 
 CONF_DIR=/etc/opt/prestoadmin
 mkdir -p $CONF_DIR
+
+CONNECTOR_DIR=$CONF_DIR/connectors
+mkdir -p $CONNECTOR_DIR
```

Comment:
Who owns these directories? root?
It's ok whatever it is, but @rschlussel should keep it in mind for her work to remove the requirement to run as sudo.